deployment/nginx.conf

```diff
@@ -0,0 +1,24 @@
+server {
+    listen 80;
+
```

Comment:
I'm slightly unclear if we also need a `server_name` statement here or not. I think I'm just going to try it on staging and see what happens.

Reply:
seems to be fine without it :shrug: